src/ray/raylet/node_manager.h

```diff
@@ -237,9 +237,12 @@ class NodeManager : public rpc::NodeManagerServiceHandler,
   /// Handler for the addition or updation of a resource in the GCS
   /// \param node_id ID of the node that created or updated resources.
   /// \param createUpdatedResources Created or updated resources.
+  /// \param labels Created or updated labels of this node.
   /// \return Whether the update is applied.
-  bool ResourceCreateUpdated(const NodeID &node_id,
-                             const ResourceRequest &createUpdatedResources);
+  bool ResourceCreateUpdated(
+      const NodeID &node_id,
+      const ResourceRequest &createUpdatedResources,
+      const absl::flat_hash_map<std::string, std::string> &labels = {});
```

Comment:
can we have a separate method for updating labels since `ResourceCreateUpdated` is supposed to only update resources

Reply:
Fixed